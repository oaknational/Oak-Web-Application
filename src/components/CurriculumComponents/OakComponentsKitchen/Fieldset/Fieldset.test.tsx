import { Fieldset, FieldsetLegend } from ".";

import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";

describe.skip("Fieldset", () => {
  test("render", async () => {
    const { baseElement } = renderWithTheme(
      <Fieldset>
        <FieldsetLegend>testing</FieldsetLegend>
      </Fieldset>,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
