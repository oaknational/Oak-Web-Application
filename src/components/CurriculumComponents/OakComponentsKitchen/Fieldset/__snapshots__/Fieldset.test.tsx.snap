// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fieldset render 1`] = `
<body>
  <div>
    <fieldset
      class="sc-ad0ad6f5-0 sc-3dab0ad0-0 eENcGS fMxjKk"
    >
      <legend
        class="sc-imWYAI sc-3dab0ad0-1 ieWUIa fOKyMZ"
      >
        testing
      </legend>
    </fieldset>
  </div>
</body>
`;
